pxfish/push.py

```diff
@@ -82,30 +104,45 @@ def push(aq, directory, component_names):
         directory (String): Directory where files are to be found
         files_to_write (List): List of files to push
     """
-    with open(os.path.join(directory, 'definition.json')) as f:
+    with open(os.path.join(directory_path, 'definition.json')) as f:
         definitions = json.load(f)
-
+    
     for name in component_names:
         file_name = "{}.rb".format(name)
         try:
-            with open(os.path.join(directory, file_name)) as f:
+            with open(os.path.join(directory_path, file_name)) as f:
                 read_file = f.read()
 
+    # TODO; create test file if it doesn't exist?
+
         except FileNotFoundError as error:
             logging.warning(
                 "Error {} writing file {} file does not exist".format(
                     error, file_name))
-            continue
-
-        local_op_type = aq.OperationType.where(
-            {"category": definitions['category'], "name": definitions['name']})
-
+            return
+
+        user_id = aq.User.where({"login": aq.login}) 
+        
+        if name == 'source':
+            op_type_or_lib_object = aq.Library.where({"category": definitions['category'], "name": definitions['name']})
```

Comment:
use a shorter variable name.
given context where it is used perhaps `parent_object`